app/src/main/java/com/framgia/soundcloud/data/source/local/provider/ContentResolverTrack.java

```diff
@@ -6,6 +6,7 @@
 import android.database.Cursor;
 import android.net.Uri;
 import android.provider.MediaStore;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports